### PR TITLE
Correct string formatting for mappers and probabilities.

### PR DESCRIPTION
**Pull request by dhuppenkothen**
I just ran entrofy on a bunch of categories that all had "yes"/"no" answers, and ran into problems when defining the prefixes. This PR should fix it.

---
This change is [Reviewable](https://reviewable.io/reviews/dhuppenkothen/entrofy/33)

### CODE REVIEW

File: entrofy/core.py [hunk 1]
```diff
@@ -179,7 +179,7 @@ def entrofy(dataframe, n, mappers=None, weights=None, pre_selects=None,
         new_df = mapper.transform(dataframe[key])
         df_binary = df_binary.join(new_df)
         all_weights.update({k: weights[key] for k in new_df.columns})
-        all_probabilities.update(mapper.targets)
+        all_probabilities.update({'{}{}'.format(mapper.prefix, k):mapper.targets[k] for k in mapper.targets.keys()})
```
Comment (bmcfee) [inline entrofy/core.py]: not sure you need the `.keys()` here; iteration over dicts should produce keys.

File: entrofy/mappers.py [hunk 2]
```diff
@@ -77,8 +77,10 @@ def transform(self, column):
             A DataFrame with the same index as `column`, and one binary-valued
             column for each potential output.
         '''
+        new_columns =  sorted(['{}{}'.format(self.prefix, key) for key in self.targets.keys()])
```
Comment (bmcfee) [inline entrofy/mappers.py]: likewise here

**Comment by dhuppenkothen** [comment]
Fixed as per your suggestion, @bmcfee. Think I didn't break anything while doing that. :)

**Comment by dhuppenkothen** [comment]
ping @bmcfee: Somehow travis is failing on this because of some `conda` issue. Since you know a lot more than me about conda, could you take a look? We should really merge this bug fix. :)

**Comment by bmcfee** [comment]
I've seen this before, but can't remember what the exact issue is.  I can take a hack at it in a separate PR though.

**Comment by bmcfee** [comment]
Okay, if you rebase it should be fixed now.

**Comment by dhuppenkothen** [comment]
Awesome, thanks so much! Merging this.

**Pull request merged**